src/documents/consumer.py

```diff
@@ -450,6 +450,7 @@ def run_post_consume_script(self, document: Document):
             "document-thumb",
             kwargs={"pk": document.pk},
         )
+        script_env["DOCUMENT_OWNER"] = str(document.owner)
```

Comment:
` =  document.owner.get_username() if document.owner else ""` would do it, then indicate the ("if any") similar to the task id and tags

Reply:
:thumbsup: done